<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Cipher GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-10</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <widget class="QWidget" name="pageUtama">
     <widget class="QPushButton" name="pushButtonCaesar">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Caesar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonViginere">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Viginere</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelChoice">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>250</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Pilih Metode Cipher</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Metode Chiper Klasik</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Metode Algoritma Modern</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Metode Algoritma Super</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonXor">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Xor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBlock">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>220</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Block</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLineEdit" name="lineOutputBlock">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>380</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>  Hasil Akan Tampil Disini</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBlocDecrypt">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Block Cipher AES</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditInputBlock">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Masukkan teks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackBlock">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>440</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBlockEncrypt">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Round Key 0 (16 karakter)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditInputBlockKey">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Masukkan teks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSuper">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Super</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageXor">
     <widget class="QPushButton" name="pushButtonXorEncrypt">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>280</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonXorDecrypt">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditInputXor">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Masukkan teks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBackXor">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>410</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditInputXor2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Masukkan Enkripsi</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineOutputXor">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>320</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>  Hasil Akan Tampil Disini</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>XOR chiper</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageSuper">
     <widget class="QPushButton" name="pushButtonBackSuper">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageCaesar">
     <widget class="QLineEdit" name="lineEditInput">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Masukkan teks</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditShift">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Masukkan shift (angka)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEncrypt">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBack">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>360</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineOutputCaesar">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Masukkan teks</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Caesar Chiper</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageViginere">
     <widget class="QPushButton" name="pushButtonBackViginere">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>410</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditInputViginere">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Masukkan teks</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditInputViginere2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Masukkan Enkripsi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEncryptViginere">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDecryptViginere">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>280</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineOutputViginere">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>320</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>  Hasil Akan Tampil Disini</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Viginere Chiper</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
